components/collection/activity/ownerInsightsTabs/HolderTab.vue

```diff
@@ -61,15 +61,14 @@
         </div>
         <hr class="my-3 mx-5" />
       </div>
-    </template>
+    </div>
     <div
       v-else
       class="is-flex is-justify-content-center is-align-items-center pt-6 px-2">
       <div class="has-text-grey">
         {{ $t('activity.noHolders') }}
       </div>
     </div>
-    <div ref="target" />
```

Comment:
deleting this completely will break scrolling
it is still needed in the case where there is content

Reply:
Hello @prachi00 👀 👋  seems like you have unresolved convo here 👼 
I do not want to break scroll 🥺

Reply:
@vikiival I solved it, forgot to mark though